Sky Atlas/Sky Atlas.html

```diff
@@ -12537,11 +12562,16 @@ <h1>Sections & Primary Docs</h1>
         </tr>
         <tr>
           <td>
-            <dfn>A.3.3 - Near Term Treatment</dfn>
+            <dfn>A.3.3 - Near-Term Treatment</dfn>
           </td>
-          <td>Near Term Treatment</td>
+          <td>Near-Term Treatment</td>
```

Comment:
This is not hyphenated in the forum post. @aa-lex can confirm if hyphenation should be added or not.

```suggestion
          <td>Near Term Treatment</td>
```

Reply:
We can hyphenate these ones, and I will create an issue in Notion to check if we should update some more.